virt/virtual_machines/virt-accessing-vm-ssh.adoc

```diff
@@ -43,9 +43,11 @@ include::modules/virt-about-static-and-dynamic-ssh-keys.adoc[leveloffset=+2]
 
 You can add a statically managed public SSH key when you create a virtual machine (VM) by using the {product-title} web console or the command line. The key is added as a cloud-init data source when the VM boots for the first time.
 
-[TIP]
+You can also add a public SSH key to a project when you create a VM by using the web console. The key is saved as a secret and is added automatically to all VMs that you create.
+
+[NOTE]
 ====
-You can also add the key to a project by using the {product-title} web console. Afterwards, this key is added automatically to VMs that you create in the project.
+If you add a secret to a project and then delete the VM, the secret is retained because it is a namespace resource. You must delete the secret manually.
```

Comment:
```suggestion
If you add a secret to a project and then delete the VM, the secret is retained because it is a namespace resource. You must manually delete the secret.
```
In general, place an adverb before the verb it modifies.

Reply:
Thank you for the review Laura! 

Nevertheless, can I ask what your source is for this? I didn't find it in any of the style guides, and in the context of general syntactic theme-rheme structure in English, the new/emphasised information (so "manually" here) tends to be at the end of a sentence by default.

I guess it's not such a big deal, but I've been writing like this for a while now, and if I'm going against any style guides that we should adhere to, I'd like to know :-)

Reply:
Good question! That guideline isn't specific to IBM or Red Hat, so you won't find it in IBM Style or the SSG. It's just a general grammar guideline that I learned in school when I learned the parts of speech. Your question prompted me to do some research, because what if I actually have it wrong? 😅 

I found that MLA Style advises placing an adverb between the subject and the verb. That makes sense to me, as I believe I followed MLA Style long ago in school. I suppose I also always assumed that IBM Style had an unspoken agreement with that guideline, as the [Adverbs - only](https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only) topic states to place _only_ immediately before the word it modifies. In this case, _manually_ is the adverb and _delete_ is the verb it modifies, so my instinct was to move _manually_ before _delete_. Your original wording wasn't inaccurate, though, so if you'd like to keep it, feel free.